measures/UpgradeCosts/tests/upgrade_costs_test.rb

```diff
@@ -14,7 +14,9 @@ def test_SFD_1story_FB_UA_GRG_MSHP_FuelTanklessWH
       'Wall Area, Above-Grade, Exterior (ft^2)' => 196.0 + 429.0 + 166.0 * 2 + 96.0 * 2 + 18.0 + 192.0 + 292.0 + 525.0,
       'Wall Area, Below-Grade (ft^2)' => 292.0 + 525.0 + 196.0 + 429.0 + 96.0 * 2,
       'Floor Area, Conditioned (ft^2)' => 2250.0 * 2,
+      'Floor Area * Infiltration Reduction, Conditioned (ft^2 * Delta ACH50)' => (3 - 2.25) * (2250.0 * 2),
       'Floor Area, Attic (ft^2)' => 2250.0,
+      'Floor Area * Insulation Increase, Attic (ft^2 * Delta R-value)' => (61.6 - 39.3) * 2250.0,
```

Comment:
Shoot. This is a problem. We really want to use the cavity R-value, not assembly R-value, for the calculation. E.g., this should presumably be 60.0 - 40.0. Not sure if we're okay with the fact that it's close but not exact, or we need to figure out how to address this...

(Migrating this comment from the previous PR so we don't lose it.)

Reply:
@ejhw feels like this is probably close enough, and that there's probably enough uncertainty in the cost values anyway.

Options:
(1) Or do we have, or could we create, methods for backing out cavity R-value from assembly R-value? @shorowit will look into this.
(2) Halfway approach. Subtract air film.
(3) Create new argument for cavity R-value and specify it in the lookup. Then we could use it here.

Let's go with (3).